src/Entity/File.php

```diff
@@ -11,6 +11,26 @@
  */
 class File extends Entity
 {
+    /**
+     * Status value for a File that has been newly uploaded, not yet proccessed by "Filer".
+     */
+    const FILE_UPLOADED = 'uploaded';
+
+    /**
+     * Status value for a File that has been deleted. (File still exists on disk!)
+     */
+    const FILE_DELETED = 'deleted';
+
+    /**
+     * Status value for a File that has been proccessed.
+     */
+    const FILE_DONE = 'done';
```

Comment:
I think processed would be a better name for it, because all other names are action names